powerlevel9k.zsh-theme

```diff
@@ -1130,14 +1130,24 @@ build_test_stats() {
   (( ratio < 50 )) && "$1_prompt_segment" "$2_BAD" "$3" "red" "$DEFAULT_COLOR" "$headline: $ratio%%" "$6"
 }
 
+# System date
+prompt_date() {
+  local date_format "%D{%d.%m.%y}"
+  if [[ -n "$POWERLEVEL9K_DATE_FORMAT" ]]; then
+    date_format="$POWERLEVEL9K_DATE_FORMAT"
+  fi
```

Comment:
Another way would be `set_default POWERLEVEL9K_DATE_FORMAT "%D{%d.%m.%y}"`, and use `$POWERLEVEL9K_DATE_FORMAT` directly. ;)